### DISCUSSION TITLE
0.35.0

### DISCUSSION THREAD

**Discussion by thijsvanloef** [thijsvanloef/palworld-server-docker#522] [Announcements]
## ⚠️Breaking Changes⚠️

**If you are planning on running Palworld version 0.2.0.6. please update to 0.35.0 as soon as possible.**

🛠️Update environment variables to include new 0.2.0.6. options
🛠️Change PalServer-Linux-Test to PalServer-Linux-Shipping in all pidof calls (prevented server from shutting down nicely)
🛠️Change PalServer-Linux-Test to PalServer-Linux-Shipping to fix ARM64 based servers not booting after updating to 0.2.0.6.

NOTE: I will have a look at the other problems ASAP, however these problems were highest priority.

## What's Changed
* Bump golang from 1.22.1-alpine to 1.22.2-alpine by @dependabot in https://github.com/thijsvanloef/palworld-server-docker/pull/516
* Fixes for update 0.2.0.6 by @thijsvanloef in https://github.com/thijsvanloef/palworld-server-docker/pull/519

**Full Changelog**: https://github.com/thijsvanloef/palworld-server-docker/compare/0.34.0...0.35.0

<hr /><em>This discussion was created from the release <a href='https://github.com/thijsvanloef/palworld-server-docker/releases/tag/0.35.0'>0.35.0</a>.</em>

**Comment by MightyBobo** [comment]
Anyone else suddenly unable to connect to their server after this?  Pulled the latest version, everything SEEMS fine, but my connection attempts time out...

**Reply by M0nsieurT** [reply]
If you want to check your server status you can type this command:
`docker exec -it palworld-server rcon-cli Info`
It will give you server information or it will tell you "connection refused", which means your server cannot boot after update.
In that case type the command below:
`docker logs --follow palworld-server`
It will give you the logs explaining why it is rebooting

**Reply by MightyBobo** [reply]
Appreciate the response. I just left for an out of town trip but I'll troubleshoot when I return next week and share results.

**Reply by MightyBobo** [reply]
@M0nsieurT Finally made it home to begin troubleshooting this.  Running the exec command, it appears my server is still running v.0.1.5.1, but I have removed the docker container and done a fresh pull on the latest from here.

The logs appear to be pretty much the same as before the major update, nothing seems to stick out.  I try to connect w/ my client, and I see nothing happening in the log at all.  It just times out on my client.

Checking Docker processes, the correct port (8211) appears to be open just fine, nothing changed there.  I'm at a loss here...

**Reply by MightyBobo** [reply]
Progress.  I changed the data location in my compose file (used to be ./data:/palworld/, now it is ./palworld:/palworld).

This successfully pulled a brand new server down and when I ran the command it showed 2.1.0.  So I'm guessing that for some reason it's not actually updating the server version locally?  Perhaps I need to change stuff further down the tree?

**Reply by MightyBobo** [reply]
Got it!  I made a backup of the steamapps folder just in case, and removed the existing one.  Server is back up.  Your commands helped me figure out the versioning issue, so I appreciate the help!

**Reply by M0nsieurT** [reply]
You are very welcome ! Docker containers can be tricky to handle for beginners... Have fun :)
